blog/03-08-2023-a-comparison-of-hardware-software-and-human-oracles/a-comparison-of-hardware-software-and-human-oracles.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A Comparison of Hardware, Software and Human Oracles
+description: In this article, we will examine the differences between Human, Hardware and software Oracle
+authors:
+  - name: ✍️ Boyejo Oluwafemi
+url: https://github.com/boyfem
+image_url: https://github.com/boyfem
+tags: [beginner, celosage, celo]
+hide_table_of_contents: true
+slug: /tutorials/a-comparison-of-hardware-software-and-human-oracles
+---
+
+![a-comparison-of-hardware-software-and-human-oracles](https://user-images.githubusercontent.com/47746690/223860476-1f130734-99f5-4661-8bd6-004d1b4a9a5e.png)
```

Comment:
We are supposed to use the new header image, which is updated in the trello card, and make sure the header image is stored in data-tutorials/intermediate or beginner or advanced whichever suits your article, Thank you

Reply:
Image is correct but , you have to use like that ![header](../../src/data-tutorials/showcase/beginner/a-comparison-of-hardware-software-and-human-oracles.png)

Reply:
../../src/data-tutorials/showcase/beginner/a-comparison-of-hardware-software-and-human-oracles.png